proto/v1/workspace_service.proto

```diff
@@ -18,4 +19,25 @@ service WorkspaceService {
       body: "*"
     };
   }
+
+  rpc PatchIamPolicy(PatchIamPolicyRequest) returns (IamPolicy) {
```

Comment:
A temp method right?

Reply:
I'm considering using it in the frontend. Post the full IAM policy body could cause inconsistent data

Reply:
There is etag that we can use to prevent inconsistent data later. Also, there are not a lot of admins changing policies so I am not too worried.

https://google.aip.dev/154

Reply:
We can add an `etag` field in `SetIamPolicyRequest` for that.

Reply:
OK let's treat it as a temp method. We can consider the etag after migration.